frontend/public/components/nav/nav-header.tsx

```diff
@@ -55,25 +99,52 @@ const NavHeader_: React.FC<NavHeaderProps & StateProps> = ({ onPerspectiveSelect
     [isPerspectiveDropdownOpen, togglePerspectiveOpen],
   );
 
-  const perspectiveItems = React.useMemo(
-    () =>
-      perspectiveExtensions.map((nextPerspective: Perspective) => (
+  const perspectiveItems = React.useMemo(() => {
+    const items = perspectiveExtensions.map((nextPerspective: Perspective) => (
+      <DropdownItem
+        key={nextPerspective.properties.id}
+        onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
+          onPerspectiveSelect(event, nextPerspective)
+        }
+        isHovered={nextPerspective.properties.id === activePerspective}
+        component="button"
+      >
+        <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
+          <span className="oc-nav-header__icon">{nextPerspective.properties.icon}</span>
+          {nextPerspective.properties.name}
+        </Title>
+      </DropdownItem>
+    ));
+    if (acmLink) {
+      const acmPerspective: Perspective = {
+        type: 'Perspective',
+        properties: {
+          id: ACM_LINK_ID,
+          name: 'Advanced Cluster Management',
+          icon: <CubesIcon />,
+          getLandingPageURL: () => acmLink.spec.href,
+          getK8sLandingPageURL: acmLink.spec.href,
+          getImportRedirectURL: acmLink.spec.href,
+        },
+      };
+      items.push(
         <DropdownItem
-          key={nextPerspective.properties.id}
+          key={acmPerspective.properties.id}
           onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
-            onPerspectiveSelect(event, nextPerspective)
+            onPerspectiveSelect(event, acmPerspective)
```

Comment:
I think it's cleaner to have a separate event handler for ACM instead of having the `if` logic above in `onPerspectiveSelect`.

Reply:
Hi @spadgett, you sure?  The new click handler would need everything in the original handler.  I think the duplication would be greater than the `if` logic.

Reply:
It could simply be

```
onClick={() => window.location.href = acmLink.spec.href}
```

which is really no duplication. I think it's a little cleaner this way. You might not need to create an `acmPerspective` item this way, just add the `DropdownItem` with its own handler.

Reply:
thanks, fixed.